doc/developers-faqs.md

```diff
@@ -58,6 +58,8 @@ you can disable the automatical update of the tags by adding its CMake name in t
 * On the branch `releases/yyyy.mm` modify the default value of the `ROBOTOLOGY_PROJECT_TAGS` CMake option to be `Custom` and of the `ROBOTOLOGY_PROJECT_TAGS_CUSTOM_FILE` to the point to the `yyyy.mm.yaml` file.
 * Create a new tag and release `vyyyy.mm` on the  `releases/yyyy.mm` branch
 
+Now all those steps have been automated by a [Github Action](https://github.com/robotology/robotology-superbuild/actions/workflows/release.yml), just manually trigger the workflow specifying the version of the new release.
```

Comment:
Nitpick: `Now` is truth now, but it may sound strange in the future. I think it would be better to put this sentence at the beginning at this `## How to do a new release` section.

Reply:
Ok ! Feel free to push on this branch and squash the commits